js/capture.js

```diff
@@ -135,7 +135,7 @@ const capture = {
         target: targetUrl.hostname,
         origin: originUrl.hostname,
         requestTime: response.timeStamp,
-        firstParty: false
+        firstParty: 0
```

Comment:
I think this should remain boolean here and changed in store.js. Perhaps in store.js add a comment that indexes can't be on boolean when you make that conversion too?